code/string_algorithms/src/remove_consecutive_duplicates/stringRemoveDuplicates.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+#include <string>
+std::string stringRemoveDuplicates(std::string s)
+{
+    std::string s1;
+    s1 += s[0];
+    for (int i = 1; i < s.length(); i++) {
+        if (s[i] != s[i - 1])
+            s1 += s[i];
+    }
+    s = s1;
+    return s;
+}
+int main()
+{
+    std::string s, s1;
+    std::cin >> s;
+    s1 = stringRemoveDuplicates(s);
```

Comment:
Declare `s1` on this line.

Reply:
done sir plzz review

Reply:
SIr I have fixed the issues plzz review

Reply:
Sir I have fixed the issue plzz review..